Assets/UI/Panels/citypanel.lua

```diff
@@ -107,6 +114,7 @@ function CQUI_CityviewEnableManager()
     LuaEvents.CQUI_WorldInput_CityviewEnable();
 end
 
+-- ===========================================================================
```

Comment:
I am not a fan of method separators like this one, but you are the one supporting his code, for the most part. if you find this convenient - sure, why not :)

Reply:
visual separators are good things (at least for me) :smile: